packages/frontend/src/features/errorBoundary/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,50 @@
+import { Flex, Text } from '@mantine/core';
+import { Prism } from '@mantine/prism';
+import * as Sentry from '@sentry/react';
+import { IconAlertCircle } from '@tabler/icons-react';
+import React, {
```

Comment:
you can remove this unused import of `React`
```suggestion
import {
```